scalahost/src/main/protobuf/semanticdb.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package org.scalameta.semantic.v1.proto;
```

Comment:
I suggest `scala.meta.internal.semantic.v1.protobuf`. `org.scalameta` is typically used for functionality reusable between projects inside `scalameta/*`.